internal/core/validations/validations.go

```diff
@@ -74,6 +74,31 @@ func IsMaxSurgeValid(maxSurge string) bool {
 	return true
 }
 
+// IsPdbMaxUnavailableValid check if PdbMaxUnavailable is valid. A PdbMaxUnavailable valid is either
+// a string that can be converted to a number greater than 0, or a string with percentage that its
+// value is greater than 0 and less than 100. Empty strigs are valid, we'll use the defualt value on
+// SchedulerManager
+func IsPdbMaxUnavailableValid(pdbMaxUnavailable string) bool {
```

Comment:
why duplicate this validation here?

Reply:
At first, it was because at one point we're validating the arguments that come from the HTTP API and at the other, we want to validate what we're reading from the Viper config. But I guess I can just call the same function when reading Viper config

Reply:
I've refactored to use the same validation function and removed this one 🙏🏽